### PR TITLE
QSI Vocab Updates

### PR DESCRIPTION
**Pull request by annegoldsack**
Updates to three QSI vocabs

### CODE REVIEW

File: vocabularies-qsi/go-categories.ttl [hunk 1]
```diff
@@ -4758,1087 +5896,1296 @@ cs:timber-operations
...
+    schema:creator
+        [
             a schema:Organization ;
-            schema:name "Place Names Working Group" ;
-            schema:url ""^^xsd:anyURI
+            schema:name "ICSM Place Names Working Group" ;
+            schema:url ""^^xsd:anyURI ;
         ] ;
+    schema:dateCreated "2015-01-01"^^xsd:date ;
+    schema:dateModified "2024-06-11"^^xsd:date ;
```
Comment (edmondchuc) [inline vocabularies-qsi/go-categories.ttl:7179]:
```suggestion
    dcterms:dateCreated "2015-01-01"^^xsd:date ;
    dcterms:dateModified "2024-06-11"^^xsd:date ;
```

File: vocabularies-qsi/go-categories.ttl [hunk 2]
```diff
@@ -1,2878 +1,3519 @@
 PREFIX cs: <https://linked.data.gov.au/def/go-categories>
-PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX go: <https://linked.data.gov.au/def/go-categories/>
```
Comment (edmondchuc) [inline vocabularies-qsi/go-categories.ttl:2]:
```suggestion
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX go: <https://linked.data.gov.au/def/go-categories/>
```

**Review by edmondchuc — CHANGES_REQUESTED** [review]
Added suggested changes which should hopefully fix the failing validator test.

**Review by edmondchuc — APPROVED** [review]

**Comment by edmondchuc** [comment]
@Steven-Bowden, Anne has made changes to a few QSI vocabs. We have fixed the issues raised by the validator.

Can you tell us what are the next steps to get this merged in and available online in Prez? Thanks!

**Review by Steven-Bowden — APPROVED** [review]
looks fine

**Review by GSQ-AI — APPROVED** [review]

**Pull request merged**